pubpub/pubpub
# Add submissions to scope summary

addresses #1910

Updates scope summaries to reflect the number of `received`-status submissions.

_Test plan_

1. Go to a collection which has submissions enabled (for example, [brand new spankin'](http://localhost:9876/dash/collection/brand-new-spankin/overview))
2. Observe count of submissions in the scope summary on the right
3. Go through usual submission process to make a new spub
4. Observe new count in submission overview (pubs and submissions should have incremented)
5. Navigate to dash for new pub
6. Delete new pub
7. Observe new count in submission overview (pubs and submissions should have decremented)

![Screenshot from 2022-04-12 16-07-06](https://user-images.githubusercontent.com/14115194/163047360-580ed0f0-65bc-44db-90db-9f7eb09d613b.png)
